feat: Option to use toImageSync in ImageComposition class (#2593)
flame-engine/flame: doc/flame/rendering/images.md

    Vector2(128, 0),
    source: Rect.fromLTWH(32, 32, 64, 64),
  );
  
Image image = await composition.compose();
Image imageSync = composition.composeSync();
```

As you can see, two versions of composing image are available. Use `ImageComposition.compose()` for

> old async approach. Or switch to new `ImageComposition.composeSync()` function to rasterize the

Reviewer: ```suggestion
the async approach. Or use the new `ImageComposition.composeSync()` function to rasterize the
```

> image into GPU context using benefits of `Picture.toImageSync` function.

Reviewer: ```suggestion
image into GPU context using the benefits of `Picture.toImageSync` function.
```

> image into GPU context using the benefits of `Picture.toImageSync` function.

Reviewer: ```suggestion
image into GPU context using the benefits of the `Picture.toImageSync` function.
```
